### PR TITLE
[INTRO] Added intro + refs

### PR DESCRIPTION
**Pull request by MattSScott**
I think it encapsulates the key aspects of this project:

1.) intro to self-org + CPR + LPG
2.) problems offered by the project (no monitoring/central auth)
3.) possible workarounds (treaties)

### CODE REVIEW

File: 000_introduction/000_introduction.tex [hunk 1]
```diff
@@ -1,8 +1,14 @@
...
+
+The absence of a centralised authority, coupled with an inability to form self-governing institutions proposes a seemingly unsolvable problem: a common pool of resources must be maintained across a network of players, each acting rationally to maximise local utility \cite{oberUtil}, without the prospect of forming enforceable rules. This necessitates the presence of a form of self-organisation sufficiently powerful as to overcome the challenges set by the nature of this problem, whilst accepting the risk that such a self-organising mechanism may act perniciously under these circumstances \cite{dobSteg}.
+
+Treaties act as a form of institutionalised power between players, serving both ``as a juristic act and as a rule'' \cite{reuter1995introduction}, with society coming to a mutual agreement upon the importance of such a legal device. Overall, it is the topic of this research to evaluate if the use of treaties as a means of self-organisation can not only combat the difficulties poised by such an unconventional CPR game, but achieve a level of stability if this game is played iteratively in an \textit{N player} scenario.
 
 \section{Our Goals}
```
Comment (mdubied) [inline 000_introduction/000_introduction.tex]: Do we need to add some goals? If yes, we could add very high-level goals such as:

- build a piece of software capable of simulating the given scenario
- design agents that interact with their environment and with each other, to eventually form a self-organised multi-agent system
- using the infrastructure and the proposed agents, perform experiments using different configuration, and analyse the results
- maybe list a few theoretical question we would like to answer, for example "Can this system reach a form of stability by using treaties?", etc

File: 000_introduction/000_introduction.tex [hunk 2]
```diff
@@ -1,8 +1,14 @@
 \chapter{Introduction}\label{introduction}
 
-\section{Problem Statement}
-Section on the tower and how we have interpreted the spec.
-Should include a basic spec for features that we wanted to achieve
+Co-operative survival games refer to a subset of political choice games wherein `players' must work together to overcome disaster, else suffer the consequences through both personal and communal damage. Instances of co-operative survival exist in varying media: computer games (\emph{e.g.} ``Don't Starve,'' ``Rust,'' ``Minecraft''), board games (e.g. Ravine) or sociological constructs (e.g. Reducing Viral Transmission ) all of which exhibit a necessity for collective action, with such notions of survival supported by extensive literature \cite{su10030652}. Survival games are often identified within the context of ``Common Pool Resource Management'' (CPR) \cite{ostromCollectiveAction} \cite{smartHome} problems, where to facilitate co-operative survival, a pool of shared resources must be maintained to help mitigate disaster. In the general case of such survival games, following a disaster, if 1 player dies, all players die.
```
Comment (mdubied) [inline 000_introduction/000_introduction.tex]: uniformity of the "e.g.", choose the option we decide together on slack.

Comment (mdubied) [inline 000_introduction/000_introduction.tex]: Are we happy with this reference citation style? I'm personally ok with it, but want to raise the question.

File: 000_introduction/000_introduction.tex [hunk 3]
```diff
@@ -1,8 +1,14 @@
...
+
+The absence of a centralised authority, coupled with an inability to form self-governing institutions proposes a seemingly unsolvable problem: a common pool of resources must be maintained across a network of players, each acting rationally to maximise local utility \cite{oberUtil}, without the prospect of forming enforceable rules. This necessitates the presence of a form of self-organisation sufficiently powerful as to overcome the challenges set by the nature of this problem, whilst accepting the risk that such a self-organising mechanism may act perniciously under these circumstances \cite{dobSteg}.
+
+Treaties act as a form of institutionalised power between players, serving both ``as a juristic act and as a rule'' \cite{reuter1995introduction}, with society coming to a mutual agreement upon the importance of such a legal device. Overall, it is the topic of this research to evaluate if the use of treaties as a means of self-organisation can not only combat the difficulties poised by such an unconventional CPR game, but achieve a level of stability if this game is played iteratively in an \textit{N player} scenario.
```
Comment (mdubied) [inline 000_introduction/000_introduction.tex]:
```suggestion
Treaties act as a form of institutionalised power between players, serving both ``as a juristic act and as a rule'' \cite{reuter1995introduction}, with society coming to a mutual agreement upon the importance of such a legal device. Overall, it is the topic of this research to evaluate if the use of treaties as a means of self-organisation can not only combat the difficulties poised by such an unconventional CPR game, but achieve a level of stability if this game is played iteratively in an \textit{N players} scenario.
```
N player**s** ?

**Review by mdubied — CHANGES_REQUESTED** [review]

**Comment by mdubied** [comment]
Happy with the content of the  introduction, good job @MattScottEEE !
The problem is presented in a very theoretical way, but we do not give any information about what is the actual setup (the platform problem). We might keep it like that, but in this case, we have to make sure there is a sufficiently good description of the setup in the first few chapters (simulation structure probably)

**Comment by priyachh** [comment]
> Happy with the content of the introduction, good job @MattScottEEE ! The problem is presented in a very theoretical way, but we do not give any information about what is the actual setup (the platform problem). We might keep it like that, but in this case, we have to make sure there is a sufficiently good description of the setup in the first few chapters (simulation structure probably)

I think we can have a brief scenario section about the actual set up? A non technical paragraph just outlining the problem like it was done in the presentation.

File: 000_introduction/000_introduction.tex (hunk 3 above)
Comment (jzzheng22) [reply 000_introduction/000_introduction.tex]: N player is correct here

**Comment by udai-arneja** [comment]
Hi Matt, great job as a basis. I do think we need to have so more information directly related to the project & what the aims are etc. Maybe a little more context about the problem before some of the information about the treaties etc - I think its worth a discussion to combine this with my "Decomposition of The Platform Simulation" to make a really awesome setup. All the content here is definitely useful, just perhaps not the right place.

**Comment by MattSScott** [comment]
Hi all:

Generally speaking, the abstract will introduce the problem we're looking to tackle. The intro is merely a means of introducing the background to the technical aspects of the project (definitions of self organisation, say)

The scenario should contextualise the problem as a platform 

@udai-arneja, @priyachh @mdubied appreciate the helpful comments. Let's try and schedule a meeting ASAP so I have time to edit

File: 000_introduction/000_introduction.tex (hunk 2 above)
Comment (MattSScott) [reply 000_introduction/000_introduction.tex]: As long we all use /cite{}, we can change the style at the very end of the doc :))

**Review by udai-arneja — DISMISSED** [review]
Last line/para looking better imo. After the discussion about the purpose of the intro and abstract I am happy with this, comprehensive.

**Comment by udai-arneja** [comment]
We also discussed the idea of project aims/goals which will be mentioned in the abstract and then later on it the scenario and simulation section.

**Comment by MattSScott** [comment]
> We also discussed the idea of project aims/goals which will be mentioned in the abstract and then later on it the scenario and simulation section.

Definitely! Thanks for the helpful meeting/review earlier - will submit the abstract asap

File: 000_introduction/000_introduction.tex [hunk 4]
```diff
@@ -1,8 +1,11 @@
...
+
+Apropos of Artificial Societies, to combat the difficulties poised by such an unconventional CPR game we introduce a notion of self-organisation through governance in the form of treaties. Serving both ``as a juristic act and as a rule'' \cite{reuter1995introduction}, treaties act as a form of institutionalised power between players, with society coming to a mutual agreement upon the importance of such a legal device. Overall, it is the topic of this research to evaluate if the use of treaties is sufficient for self-organising this system to achieve a level of stability when this game is played iteratively in an \textit{N-player} scenario.
```
Comment (MattSScott) [inline 000_introduction/000_introduction.tex]:
```suggestion
Apropos of Artificial Societies, to combat the difficulties poised by such an unconventional CPR game we introduce a notion of self-organisation through governance in the form of treaties. Serving both ``as a juristic act and as a rule'' \cite{reuter1995introduction}, treaties act as a form of institutionalised power between players, with society coming to a mutual agreement upon the importance of such a legal device. Overall, it is the topic of this research to evaluate if the use of treaties is sufficient for self-organising this system, so as to achieve a level of stability when this game is played iteratively in an \textit{N-player} scenario.
```

**Review by udai-arneja — APPROVED** [review]

**Review by mdubied — APPROVED** [review]
I'm happy with that! It seems that you will introduce the goals and the actual setup (aka the platform problem) in another section, so that's fine!

**Comment by MattSScott** [comment]
thanks all for the helpful comments

**Pull request merged**